test/src/test/java/hudson/util/ProcessTreeKillerTest.java

```diff
@@ -1,21 +1,35 @@
 package hudson.util;
 
+import static org.junit.Assert.*;
+import static org.junit.Assume.*;
+import java.io.File;
 import hudson.Functions;
 import hudson.model.FreeStyleBuild;
 import hudson.model.FreeStyleProject;
 import hudson.tasks.Maven;
 import hudson.tasks.Shell;
+import hudson.util.ProcessTreeRemoting.IOSProcess;
+import org.junit.After;
 import org.junit.Rule;
 import org.junit.Test;
 import org.jvnet.hudson.test.ExtractResourceSCM;
 import org.jvnet.hudson.test.Issue;
 import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.TestExtension;
+import com.google.common.collect.ImmutableMap;
 
 public class ProcessTreeKillerTest {
 
     @Rule
     public JenkinsRule j = new JenkinsRule();
-
+    private Process process;
+    
+    @After
+    public void tearDown() throws Exception {
+        if (null != process)
```

Comment:
Probably it could be just handled in a <code>finally</code> block without external fields. Just to keep things simple